.github/dependabot.yml

```diff
@@ -1,286 +0,0 @@
----
-version: 2
-updates:
-  - package-ecosystem: github-actions
```

Comment:
Lets keep the github actions. I think that would be useful.

Reply:
I updated it to just remove the got bits